package.json

```
@@ -1,19 +1,23 @@
 {
   "name": "putter",
   "version": "0.1.0",
-  "main": "main.js",
   "scripts": {
-    "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
-    "test:unit": "vue-cli-service test:unit",
-    "electron": "vue-cli-service build && electron ."
+    "electron:build": "vue-cli-service electron:build",
+    "electron:serve": "vue-cli-service electron:serve",
+    "postinstall": "electron-builder install-app-deps",
+    "postuninstall": "electron-builder install-app-deps",
```

Should this be `uninstall-app-deps` intead?
```suggestion
    "postuninstall": "electron-builder uninstall-app-deps",
```

---

Good question. I didn't add those; they were added by the electron-builder installation. I don't actually know what they do.

---

It's probably fine then